addons/account/tests/common.py

```diff
@@ -9,7 +9,6 @@
 from lxml import etree
 
 
-@tagged('post_install', '-at_install')
```

Comment:
Wouldn't it make more sense to leave it on the common class, but remove it from the extending classes?